Rocc/Manufacturer Implementations/Sony/PTP IP Camera/CameraEvent+SonyPTPIP.swift

```diff
@@ -39,11 +39,13 @@ enum SonyStillCaptureMode: DWord, SonyPTPPropValueConvertable {
     case single = 0x00000001
     case continuousHighPlus = 0x00018010
     case continuousLow = 0x00018012
-    case continuous = 0x00018013
+    case continuous = 0x00018015
     case continuousS = 0x00018014
-    case continuousHigh = 0x00018015
-    case timer10_a = 0x00010002
```

Comment:
This was found by reverse-engineering the Sony app... I basically returned values from our mocker within a certain range and noted which ones had icons in the Sony app... can we keep this in just to be safe?

Reply:
Ohh wait, never mind just seen the name has simply changed!